.travis.yml

```diff
@@ -1,23 +1,37 @@
+sudo: required
+services:
+  - docker
 language: c
-install: wget https://raw.githubusercontent.com/avsm/ocaml-travisci-skeleton/mirage3/.travis-mirage.sh
-script: bash -ex .travis-mirage.sh
+cache: false
+
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get install -y realpath # needed on ubuntu 14.04.5 LTS / trusty
+  - wget https://raw.githubusercontent.com/mor1/dommage/master/dommage
```

Comment:
It's a shell script, the universal language for running things and manipulating files. So sue me ... ;)

(In all seriousness: avoiding the need to get OPAM/OCaml correctly installed on the host Ubuntu image, and instead being able to contain all the OCaml-ness inside the pre-built and reused container seemed to simplify things to me.)